### DISCUSSION TITLE
Symbol Units Not Working

### DISCUSSION THREAD

**Discussion by HFSimas** [joaomlourenco/novathesis#229] [Q&A]
Hello,

After updating to version 6.5.2 the command `\setglossarystyle{altlong4col}` doesn't work anymore, giving the error "Package glossaries-extra Error: Glossary style `altlong4col' undefined." Is there a new way to have the units of symbols?

Also, is there a way to remove the page number where the symbols/acronyms are used?

Thank you

**Comment by joaomlourenco** [comment]
Try  `\setglossarystyle{xltabular4col}` and let me know if it works for you.

**Reply by HFSimas** [reply]
It solves the problem of the symbols but also changes the acronyms list, so bigger descriptions may get out of the margins, such as shown in the image. Is there a way to have different types of glossaries instead of defining them equally?

![image](https://user-images.githubusercontent.com/92308623/136856188-ef9f288b-63f2-4930-a99e-3aa07eacdda2.png)

Also, is there a way to not show the number of the page where symbols/acronyms are used.

Thank you.

**Reply by joaomlourenco** [reply]
@HFSimas did you solve your issues.  Do you still need some help?

**Reply by HFSimas** [reply]
I did not solve it yet, the same problem persists.

**Reply by HFSimas** [reply]
I saw [this thread](https://tex.stackexchange.com/questions/222772/multiple-glossaries-with-different-styles) which has the solution to use different glossary styles but wasn't able to implement it.

**Comment by joaomlourenco** [comment]
Can you please upgrade to the latest version of the template.  I believe all your problems reported here will be gone. :)

Follow these notes:
https://github.com/joaomlourenco/novathesis/wiki/upgrading-novathesis#how-to-convert-from-v6x-to-a-more-recent-v6x

**Reply by joaomlourenco** [reply]
I advise you rename your config folder into `Config-OLD` and them:
- Overwrite the `Chapters` folder with your own `Chapters` folder.
- Go through  the files in the `Config` folder and copy-in the info from the corresponding file in `Config-OLD`.  Please **copy-in carefully**, as there are some syntax changes in the new files.

**Reply by HFSimas** [reply]
I updated to the latest version 6.9.5 but the problem persists. If I use `\setglossarystyle{xltabular4col}` the units appear in the symbol list but the acronym list goes outside of the margin in the same way shown in the previous picture I sent. If I do not put that command, the units in the symbol list don't appear.

**Reply by joaomlourenco** [reply]
But why are you forcing "\setglossarystyle{xltabular4col}"?  I now believe that's the problem, because you have just two columns (the symbol and the description).  Doesn't it work without the "\setglossarystyle{xltabular4col}"??

**Reply by HFSimas** [reply]
What I would like to have in the Symbol chapter are 3 columns, the 1st with the symbol, such as B. The 2nd with the description, in this case Magnetic Flux Density, and the 3rd column with the units used, in this case T for Tesla.

My problem is that if I do not use `\setglossarystyle{xltabular4col}` the 3rd column does not appear. But if I use it, the Symbol chapter shows all 3 columns as I wanted (as shown in the image below), but the Acronym chapter goes outside of the margins.

![image](https://user-images.githubusercontent.com/92308623/154095904-8fbe9687-411f-4cec-8f76-9f31a746bc06.png)


In Symbols I'm defining each entry as `\glsxtrnewsymbol[ description = { Magnetic field intensity }, symbol = { \si{ \ampere\per\meter } } ] {H} {\( H \)}`

**Reply by joaomlourenco** [reply]
Oh!  I see now.   I'll be back to this problem asap (hopefully next week).
